tests/test_client.py

```diff
@@ -65,5 +68,6 @@ def test_event_workflow_lifecycle(client, sequential_workflow, my_event):
 
 
 def test_url_params_encoding(client):
+    client.app_env = 'prod'
```

Comment:
Because of my change of not requiring any app_env, I need to add an app env here so that the URL adds an `&`.